.travis.yml

```diff
@@ -156,9 +156,10 @@ jobs:
       before_install: mkdir -p $HOME/.bin; export PATH=$PATH:$HOME/.bin; .travis/before_install.sh
       install: .travis/install.sh
       script:
-        - RAIDENVERSION=$TRAVIS_TAG make bundle-docker
+        - . .travis/set_tag.sh
+        - make bundle-docker
         # Make sure the produced binary works
-        - dist/raiden-${TRAVIS_TAG}-linux smoketest
+        - dist/raiden-${RAIDENVERSION}-linux smoketest
```

Comment:
This will not work since the produced binary will have a name similar to `raiden-v0.7.1.dev37+g3d61c3db-macOS` (i.e. only the zip archive will have the `$RAIDENVERSION` part in it).

But this was broken by last commit anyway since the binary isn't available outside the build container on linux.

Please replace this line with:
```
- tar -xvzf -C dist raiden-${RAIDENVERSION}-linux.tar.gz
- dist/raiden-*-linux smoketest
```